.travis.yml

```diff
@@ -6,9 +6,17 @@ cache:
   directories:
     - dependencies
     - build/libs
+    - $HOME/AppData/Local/Temp/chocolatey
+    - /c/tools/msys64
 
 before_cache:
   - ${TRAVIS_BUILD_DIR}/scripts/cleanup-cmake.sh
+  - |
+    case $TRAVIS_OS_NAME in
+      windows)
+        $SHELL_CMD pacman --sync --clean --noconfirm
```

Comment:
is SHELL_CMD something you defined or is that a predefined variable

Reply:
It's defined in the install script for the MSYS build. It's undefined/empty for the other builds.